include/mitsuba/core/object.h

```diff
@@ -115,6 +115,13 @@ class MTS_EXPORT_CORE Object {
      */
     virtual std::string to_string() const;
 
+    /**
+     * \brief Suspend or resume tracking of derivatives of the object and its children.
+     *
+     * This allows to stop keeping track of gradients while not recomputing intermediate values (e.g. normals, height fields...)
```

Comment:
Discussion of `normals, height fields...` is perhaps a bit specific for a documentation at the `Object` level. I'd provide a bit of context -- something like: 

> One important use case of Mitsuba 2 entails the propagation of derivatives through the full rendering process, typically via automatic differentiation. To this end, certain variants of the renderer perform all computation via Enoki's `DiffArray<T>` class, which internally builds a computation graph of all involved arithmetic. In certain situations, this is undesirable, and we need to temporarily suspend the propagation of gradients. That is the role of this function: it recursively calls `set_grad_suspended` on child objects, and it invokes `ek::set_grad_enabled` on relevant attributes.